README.md

```diff
@@ -6,7 +6,7 @@ This app script code can be added to a google sheet to run the Ambassador OS.
 
 1. You have a google sheet with a list of ambassadors with two columns, Ambassador Email Address, and Ambassador Discord Handle.
 2. You have a google sheet with a list of ambassador contributions with 4 columns: Timestamp, Email Address, Your Discord Handle, "Dear Ambassador,
-   Please add text, inputs or links to your contributions during the month of February, 2024:"
+   Please add text, inputs or links to your contributions during the month:"
```

Comment:
Note that I suggest we remove the month from the question text - it is confusing because it is not updated.